Dockerfiles/Dockerfile

```diff
@@ -14,6 +14,14 @@ RUN if [ "${USE_LOCAL}" != "true" ]; then \
         ./get_all_manifests.sh ${OVERWRITE_MANIFESTS}; \
     fi
 
+# Copy monitoring config
+COPY config/monitoring/ /opt/manifests/monitoring
+# Copy partners config
+COPY config/partners/ /opt/manifests/partners
```

Comment:
we can skip partners one, with https://github.com/opendatahub-io/opendatahub-operator/pull/1504

Reply:
> we can skip partners one, with #1504

Why for building it cannot copy just everything?

Reply:
is the question about why not make it as `COPY config/ /opt/manifests/` in Dockefile?
if so, we have more under `config` folder which no need to get into final image